src/misc.cpp

```diff
@@ -507,8 +512,8 @@ static void* aligned_large_pages_alloc_windows([[maybe_unused]] size_t allocSize
   if (!fun6(GetCurrentProcess(), TOKEN_ADJUST_PRIVILEGES | TOKEN_QUERY, &hProcessToken))
       return nullptr;
 
-  // LookupPrivilegeValueA()
-  if (fun7(nullptr, SE_LOCK_MEMORY_NAME, &luid))
+  // LookupPrivilegeValue()
```

Comment:
Shouldn't we keep the A so that we know it's ANSI? Also we can comment that this is the hardcoded value of `SE_LOCK_MEMORY_NAME` in case someone wondered why there is a magic string.

Reply:
See updated commit, I put explicit ansi string here:

```
  if (fun7(nullptr, "SeLockMemoryPrivilege", &luid))
```

I also tried in the debugger and the large pages are allocated correctly.

Reply:
Yes, I'm talking about comments, we should write:
```
// LookupPrivilegeValueA(), "SeLockMemoryPrivilege" is the ANSI expansion of SE_LOCK_MEMORY_NAME macro.
```